grid-proxy/internal/indexer/health.go

```diff
@@ -31,28 +32,25 @@ func (w *HealthWork) Finders() map[string]time.Duration {
 }
 
 func (w *HealthWork) Get(ctx context.Context, rmb *peer.RpcClient, twinId uint32) ([]types.HealthReport, error) {
-	var response types.HealthReport
-	err := callNode(ctx, rmb, healthCallCmd, nil, twinId, &response)
+	var diagnostics diagnostics.Diagnostics
+	err := callNode(ctx, rmb, healthCallCmd, nil, twinId, &diagnostics)
+	if err != nil {
+		return []types.HealthReport{}, err
```

Comment:
there are nodes that don't have this call due to update issues in zos, since they don't have this method they will not respond to the rmb call and it will timeout. They should be reported as unhealthy. Will this error return register them as unhealthy?

Reply:
yeah, a return will just print the error and continue, so if it was healthy before it will keep the state.
updated to not return on error and instead set healthy to false